### PR TITLE
Update user-settings.md

### PR DESCRIPTION
**Pull request by BeateRixen**
I added the missing information on the right drawer I was recently referring to @eniosultan .

### CODE REVIEW

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 1]
```diff
@@ -36,6 +36,14 @@ The **User** menu contains the following items:
 <td style="text-align:left">Logout</td>
 <td style="text-align:left">Logs you out from your {{< product-c8y-iot >}} account. </td>
 </tr>
+</tr>
+<tr>
+<td style="text-align:left">Revoke tokens</td>
+<td style="text-align:left">Revoking tokens logs out all users currently logged in via "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.<br>
```
Comment (rssag) [inline content/users-guide/getting-started-bundle/user-settings.md:42]: "will"-trigger :) i don't mind it here though.

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 2]
```diff
@@ -45,21 +53,22 @@ The **User** menu contains the following items:
...
 <td style="text-align:left">Version information</td>
 <td style="text-align:left">Shows release information on the {{< product-c8y-iot >}} version you are using (for example, 10.13.0.034) for both Backend and UI. Moreover shows the ID of your tenant, which might be required if you request support. Click the copy icon next to the tenant ID to copy it to the clipboard.</td>
 </tr>
 </tbody>
 </table>
 
+In the **Platform info** section, you find information on the infrastructure of your platform:
+
+- The ID of your tenant, which might be required if you request support. Click the copy icon next to the tenant ID to copy it to the clipboard.
+- Release version information for both backend and the UI.
```
Comment (rssag) [inline content/users-guide/getting-started-bundle/user-settings.md:66]:
```suggestion
- Release version information for both the backend and the UI.
```

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 3]
```diff
@@ -45,21 +53,22 @@ The **User** menu contains the following items:
...
+
+Moreover you can click **Detailed info** to download more detailed platform information.
```
Comment (rssag) [inline content/users-guide/getting-started-bundle/user-settings.md:68]: curiosity: what's in the download?

File: content/users-guide/getting-started-bundle/user-settings.md (hunk 1 above)
Comment (BeateRixen) [reply content/users-guide/getting-started-bundle/user-settings.md:42]: After the demo yesterday I expect that this entry will change anyway:-).

File: content/users-guide/getting-started-bundle/user-settings.md (hunk 3 above)
Comment (BeateRixen) [reply content/users-guide/getting-started-bundle/user-settings.md:68]: Try it out :-). As far as I know this is supposed to provide version information on further components in the future.

**Review by rssag — APPROVED** [review]

**Review by eniosultan — APPROVED** [review]

**Pull request merged**